template/SpellAction.sol

```diff
@@ -1,19 +1,16 @@
+interface ERC20 {
+    function decimals() external returns (uint8);
+}
+
```

Comment:
Let's use 
```
import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
```
Instead.  Same with in the spell.  In fact, we should probably lift all the typical collateral imports into this template.